doc/mosaic.rst

```diff
@@ -0,0 +1,30 @@
+
+.. include:: include/links.rst
+
+.. _mosaic:
+
+===============================================
+Processing and Construction of Detector Mosaics
+===============================================
+
+If the geometry of each detector in a detector array is known *a priori*,
+``PypeIt`` can construct a mosaic of the detector data for further processing,
+instead of processing the detector data individually.  This is currently the
+default approach for Gemini/GMOS only (soon we will add Keck/DEIMOS).
+
+Coordinate Conventions
+----------------------
+
```

Comment:
Maybe add a note here indicating that this is done via NGP to avoid correlated noise.

Reply:
Yeah, I've only just started this doc, but I'll definitely include that.